### PR TITLE
Update SUBMISSION-GUIDELINES.md

### PR DESCRIPTION
**Pull request by lopegor**
Hi! I added a few lines to better clarify the guide.

### CODE REVIEW

**Comment by mochet** [comment]
@bwhm @blrhc can we please get a review on this, I don't have access to request a review in this repo. It's part of bounty 9 (https://testnet.joystream.org/#/forum/threads/303?replyIdx=8)

File: SUBMISSION-GUIDELINES.md [hunk 1]
```diff
@@ -91,7 +93,7 @@ Link: some.github.link
 1. Validator
 Date: 05.02.2021-09.02.2021
 Description: I was a validator from block/era #1337/2 to #11337/10, with stash account `5myValidatorStashAccount`. (I was active in all the eras in this range, and found a total of 1000 blocks.)
-I have signed the message: "I am member id/handle `133/joystream` with `5myValidatorStashAccount`" with my membership ctrl account - `5CJzTaCp5fuqG7NdJQ6oUCwdmFHKichew8w4RZ3zFHM8qSe6`
+I have signed the message: "I am member id/handle `133/joystream` with a Validator Stash Account: `5myValidatorStashAccount` with my membership controller (root) account: `5CJzTaCp5fuqG7NdJQ6oUCwdmFHKichew8w4RZ3zFHM8qSe6`
```
Comment (blrhc) [inline SUBMISSION-GUIDELINES.md:96]:
```suggestion
I have signed the message: "I am member id/handle `133/joystream` with a validator stash account: `5myValidatorStashAccount` and my membership controller (root) account is: `5CJzTaCp5fuqG7NdJQ6oUCwdmFHKichew8w4RZ3zFHM8qSe6`
```

Comment (blrhc) [inline SUBMISSION-GUIDELINES.md:96]:
```suggestion
I have signed the message: "I am member id/handle `133/joystream` with validator stash account: `5myValidatorStashAccount`" with my membership controller (root) account: `5CJzTaCp5fuqG7NdJQ6oUCwdmFHKichew8w4RZ3zFHM8qSe6`
```

**Comment by blrhc** [comment]
Thanks for pointing this one out.

**Pull request merged**